libraries/botbuilder-ai/tests/qnaMakerDialog.test.js

```diff
@@ -441,6 +445,8 @@ describe('QnAMakerDialog', function () {
                 adapter.send('QnaMaker_TopNAnswer.json').startTest(),
                 (thrown) => thrown.message === '`suggestionsActivity` must be defined'
             );
+
+            mock.verify();
```

Comment:
Actually - `sandbox.verify()` verifies every mock in a sandbox so I tend to use that as a catch all.

Reply:
Missed this! Thanks for catching that 😄